Makefile

```diff
@@ -31,7 +31,6 @@ init: intro do-pre-init do-install-git-hooks do-run-updates do-show-commands out
 update-project: intro do-run-updates outro
 update: intro do-switch-branch do-run-updates outro
 github: intro do-checkout-pr do-run-updates outro
-git-hooks: intro do-install-git-hooks outro
```

Comment:
`do-install-git-hooks` is still in the `init` command

Reply:
Done!